### PR TITLE
Use `eglCheck` macro as an expression instead of a statement

### PR DESCRIPTION
**Pull request by ZXShady**
Applies changes that should have been in pr #3239

### CODE REVIEW

File: src/SFML/Window/DRM/DRMContext.cpp [hunk 1]
```diff
@@ -618,7 +617,7 @@ bool DRMContext::makeCurrent(bool current)
 {
     const EGLSurface surface = current ? m_surface : EGL_NO_SURFACE;
     const EGLContext context = current ? m_context : EGL_NO_CONTEXT;
-    return m_surface != EGL_NO_SURFACE && eglMakeCurrent(m_display, surface, surface, context);
+    return m_surface != EGL_NO_SURFACE && eglCheck(eglMakeCurrent(m_display, surface, surface, context));
```
Comment (ChrisThrasher) [inline src/SFML/Window/DRM/DRMContext.cpp:620]: Oh neat so you found a place where we weren't checking for errors at all.

**Review by ChrisThrasher — APPROVED** [review]

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

**Review by ChrisThrasher — APPROVED** [review]

**Review by ZXShady — COMMENTED** [review]
~~otonun_n_~~

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

**Review by vittorioromeo — APPROVED** [review]

**Pull request merged**